test/test-remote/testutils/logs.ts

```diff
@@ -40,7 +40,7 @@ export interface LogRecord {
   labels: LogRecordLabels;
 }
 
-export async function queryLoki(baseUrl: string, queryParams: URLSearchParams) {
+async function queryLoki(baseUrl: string, queryParams: URLSearchParams) {
```

Comment:
I loved seeing you removing this `export` -- I was wondering if this function is used in different contexts, and this part of the diff was providing the answer :).